meta-rauc-bbb/recipes-bsp/u-boot/files/0001-am335x_evm_defconfig-rauc.patch

```diff
@@ -0,0 +1,44 @@
+From a367dab84601052ef0059070ab712545c18a1efa Mon Sep 17 00:00:00 2001
+From: Leon Anavi <leon.anavi@konsulko.com>
+Date: Thu, 28 Mar 2024 14:43:30 +0000
+Subject: [PATCH] am335x_evm_defconfig: RAUC
+
+Upstream-Status: Inappropriate [RAUC specific]
+
+Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
+---
+ configs/am335x_evm_defconfig | 7 +++++--
+ 1 file changed, 5 insertions(+), 2 deletions(-)
+
+diff --git a/configs/am335x_evm_defconfig b/configs/am335x_evm_defconfig
+index f048e60f7f..44ef386a90 100644
+--- a/configs/am335x_evm_defconfig
++++ b/configs/am335x_evm_defconfig
+@@ -16,7 +16,7 @@ CONFIG_TIMESTAMP=y
+ CONFIG_SPL_LOAD_FIT=y
+ CONFIG_DISTRO_DEFAULTS=y
+ CONFIG_OF_BOARD_SETUP=y
+-CONFIG_BOOTCOMMAND="run findfdt; run init_console; run finduuid; run distro_bootcmd"
++CONFIG_BOOTCOMMAND="load mmc 0:1 $loadaddr boot.scr; source $loadaddr"
+ CONFIG_LOGLEVEL=3
+ CONFIG_SYS_CONSOLE_INFO_QUIET=y
+ CONFIG_ARCH_MISC_INIT=y
+@@ -47,7 +47,7 @@ CONFIG_CMD_SPL_NAND_OFS=0x00080000
+ CONFIG_SYS_I2C_EEPROM_ADDR_LEN=2
+ # CONFIG_CMD_FLASH is not set
+ CONFIG_CMD_NAND=y
+-# CONFIG_CMD_SETEXPR is not set
++CONFIG_CMD_SETEXPR=y
+ CONFIG_BOOTP_DNS2=y
+ CONFIG_CMD_MTDPARTS=y
+ CONFIG_MTDIDS_DEFAULT="nand0=nand.0"
+@@ -124,3 +124,6 @@ CONFIG_WDT=y
+ CONFIG_DYNAMIC_CRC_TABLE=y
+ CONFIG_RSA=y
+ CONFIG_LZO=y
++CONFIG_ENV_EXT4_INTERFACE="mmc"
++CONFIG_ENV_EXT4_DEVICE_AND_PART="0:1"
++CONFIG_ENV_EXT4_FILE="/boot/uboot.env"
```

Comment:
TODO:

@a3f:
This is not power-fail safe. A sudden power cut during environment write may render the system unbootable in an A/B fashion.